### PR TITLE
Add verify address to changelog

### PR DESCRIPTION
**Pull request by shjortConcordium**
## Purpose

Add verify address to changelog.

## Changes

Also added verify address doc. 
And added INS to docs missing it.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: doc/verify_address.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Verify Address
+
+Given identity index and a credential counter/account index, displays the associated account address.
```
Comment (orhoj) [inline doc/verify_address.md:3]:
```suggestion
Given an identity index and a credential counter/account index this function displays the associated account address.
```

File: doc/verify_address.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Allows the user to approve or reject the address, determing whether the app returns success or rejection.
```
Comment (orhoj) [inline doc/verify_address.md:5]:
```suggestion
Allows the user to approve or reject the address, determining whether the app returns success or rejection.
```

File: doc/verify_address.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Verify Address
```
Comment (orhoj) [inline doc/verify_address.md:1]: The documentation here should describe the preconditions that are required for this to work, e.g. how the identity keys are assumed to have been generated from the keys that can be exported form the Ledger app as well.

Comment (shjortConcordium) [reply doc/verify_address.md:1]: I've expanded the doc.

**Review by orhoj — APPROVED** [review]

**Pull request merged**